From 1de232bd5b72534489f61eb4c666b5a892120f6d Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Tue, 18 Apr 2017 14:35:36 -0400
Subject: common: Fix plugin whitelist for gdm

The plugin names don't have a gsd- prefix, so the login screen
is currently getting started with no plugins!

This commit fixes that, and as a consequence, fixes hidpi scaling
for the login screen.

https://bugzilla.gnome.org/show_bug.cgi?id=780208
---
 plugins/common/daemon-skeleton-gtk.h | 24 ++++++++++++------------
 plugins/common/daemon-skeleton.h     | 24 ++++++++++++------------
 2 files changed, 24 insertions(+), 24 deletions(-)

diff --git a/plugins/common/daemon-skeleton-gtk.h b/plugins/common/daemon-skeleton-gtk.h
index 714e48b..3bfd618 100644
--- a/plugins/common/daemon-skeleton-gtk.h
+++ b/plugins/common/daemon-skeleton-gtk.h
@@ -39,18 +39,18 @@ static GOptionEntry entries[] = {
 };
 
 static const char *gdm_helpers[] = {
-	"gsd-a11y-keyboard",
-	"gsd-a11y-settings",
-	"gsd-clipboard",
-	"gsd-color",
-	"gsd-keyboard",
-	"gsd-media-keys",
-	"gsd-orientation",
-	"gsd-power",
-	"gsd-smartcard",
-	"gsd-sound",
-	"gsd-xrandr",
-	"gsd-xsettings"
+	"a11y-keyboard",
+	"a11y-settings",
+	"clipboard",
+	"color",
+	"keyboard",
+	"media-keys",
+	"orientation",
+	"power",
+	"smartcard",
+	"sound",
+	"xrandr",
+	"xsettings"
 };
 
 static gboolean
diff --git a/plugins/common/daemon-skeleton.h b/plugins/common/daemon-skeleton.h
index dedfd61..07f8d23 100644
--- a/plugins/common/daemon-skeleton.h
+++ b/plugins/common/daemon-skeleton.h
@@ -38,18 +38,18 @@ static GOptionEntry entries[] = {
 };
 
 static const char *gdm_helpers[] = {
-	"gsd-a11y-keyboard",
-	"gsd-a11y-settings",
-	"gsd-clipboard",
-	"gsd-color",
-	"gsd-keyboard",
-	"gsd-media-keys",
-	"gsd-orientation",
-	"gsd-power",
-	"gsd-smartcard",
-	"gsd-sound",
-	"gsd-xrandr",
-	"gsd-xsettings"
+	"a11y-keyboard",
+	"a11y-settings",
+	"clipboard",
+	"color",
+	"keyboard",
+	"media-keys",
+	"orientation",
+	"power",
+	"smartcard",
+	"sound",
+	"xrandr",
+	"xsettings"
 };
 
 static gboolean

